dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScopeTest.kt

```diff
@@ -647,6 +647,7 @@ internal class RumSessionScopeTest {
                 assertThat(it.cpuVitalMonitor).isInstanceOf(NoOpVitalMonitor::class.java)
                 assertThat(it.memoryVitalMonitor).isInstanceOf(NoOpVitalMonitor::class.java)
                 assertThat(it.frameRateVitalMonitor).isInstanceOf(NoOpVitalMonitor::class.java)
+                assertThat(it.type).isEqualTo(RumViewScope.RumViewType.BACKGROUND)
```

Comment:
You should also check that when a view is started by a `startView(…)` event, the type is `FOREGROUND`